editor/editor_node.cpp

```diff
@@ -3159,7 +3159,20 @@ void EditorNode::set_addon_plugin_enabled(const String &p_addon, bool p_enabled,
 
 		// Errors in the script cause the base_type to be an empty string.
 		if (String(script->get_instance_base_type()) == "") {
-			show_warning(vformat(TTR("Unable to load addon script from path: '%s' There seems to be an error in the code, please check the syntax."), script_path));
+			show_warning(vformat(TTR("Unable to load addon script from path: '%s'. This might be due to a code error in that script. Disabling the addon at '%s' to prevent further errors."), script_path, p_addon));
```

Comment:
```suggestion
			show_warning(vformat(TTR("Unable to load addon script from path: '%s'. This might be due to a code error in that script.\nDisabling the addon at '%s' to prevent further errors."), script_path, p_addon));
```
The message is too wide:
![image](https://user-images.githubusercontent.com/2223172/117584417-1fbb6c80-b10d-11eb-810c-107709fb4885.png)

Reply:
Added a new line to shorten the length of the message, thanks.